docs/Actions.md

```diff
@@ -245,16 +246,16 @@ You can call it inside a `<BanUser>` button component as follows:
 ```jsx
 const BanUserButton = ({ userId }) => {
     const dataProvider = useDataProvider();
-    const { mutate, isLoading } = useMutation(
-        () => dataProvider.banUser(userId)
-    );
-    return <Button label="Ban" onClick={() => mutate()} disabled={isLoading} />;
+    const { mutate, isPending } = useMutation({
+        mutationFn: () => dataProvider.banUser(userId)
+    });
+    return <Button label="Ban" onClick={() => mutate()} disabled={isPending} />;
 };
 ```
 
 ## Query Options
 
-The data provider method hooks (like `useGetOne`) and react-query's hooks (like `useQuery`) accept a query options object as the last argument. This object can be used to modify the way the query is executed. There are many options, all documented [in the react-query documentation](https://tanstack.com/query/v3/docs/react/reference/useQuery):
+The data provider method hooks (like `useGetOne`) and react-query's hooks (like `useQuery`) accept a query options object as the last argument. This object can be used to modify the way the query is executed. There are many options, all documented [in the react-query documentation](https://tanstack.com/query/v5/docs/react/reference/useQuery):
```

Comment:
> There are many options, all documented [in the react-query documentation](https://tanstack.com/query/v5/docs/react/reference/useQuery):

This is not true for the `onXXX` methods, so I think we should extract their explanation

Also, we already have a section called "Success and Error Side Effects", perhaps we should reorganize this chapter?